docs/guides/libfree-gui.md

```diff
@@ -0,0 +1,51 @@
+# Library-free DIA search using the GUI
+**This tutorial was created using alphaDIA 1.8.1 - please be aware that there might be changes in your version**
+
+## 1. Prerequisites
+Make sure that you have a machine with at least 64 gigabytes of memory.
+Please download the test data for this tutorial [here](https://datashare.biochem.mpg.de/s/Nsp8CaHMBf7FHq1). We will be using replicates of label-free bulk DIA data of HeLa digests acquired on the Orbitrap Astral.
+Also make sure you have a valid alphaDIA installation including the GUI. The easiest option is the one-click installer, and a summary of all installation options can be found [here](<project:../installation.md>).
+Also ensure the right execution engine has been selected and your version is up to date.
+<img src="../_static/images/libfree-gui/initial_engine.png" width="100%" height="auto">
+
+## 2. Project Structure
+We will be performing two DIA searches: a first search for library generation and a second search for joined quantification. To accommodate this, we prepare the project directory to have a `first_pass` and a `second_pass` folder.
```

Comment:
1. You could mention here that users can change the path to the directories in the `Output Files` tab